packages/core/src/App/Components/Elements/LiveChat/use-livechat.ts

```diff
@@ -126,13 +131,15 @@ const useLiveChat = (has_cookie_account = false, active_loginid?: string) => {
     }, [history, isMounted, onHistoryChange]);
 
     useEffect(() => {
-        if (reload) {
+        if (reload && !should_disable_livechat) {
             liveChatSetup(has_cookie_account);
             setReload(false);
         }
-    }, [reload, has_cookie_account]);
+    }, [reload, has_cookie_account, should_disable_livechat]);
 
-    useEffect(() => liveChatSetup(has_cookie_account), [has_cookie_account, active_loginid]);
+    useEffect(() => {
+        if (!should_disable_livechat) liveChatSetup(has_cookie_account);
```

Comment:
```suggestion
        if (should_disable_livechat) return;
        liveChatSetup(has_cookie_account);
```
More readable this way